packages/editor/src/components/parts/start/useStartData.ts

```diff
@@ -16,6 +42,28 @@ export function useStartData(): {
 } {
   const { config, defaultConfig, initConfig, setConfig } = useConfigDataContext();
 
+  // const [isInSynch, setInSynch] = useState(true);
+
+  // const signatureEnricher = useCallback(
+  //   (signature: string) => {
+  //     if (signaturePostfix) {
+  //       return `${signature}${signaturePostfix}`;
+  //     }
+  //     return `${signature}(${config.input.params.map(param => param.type).join(',')})`;
+  //   },
+  //   [config.input.params, signaturePostfix]
+  // );
+
+  // useEffect(() => {
+  //   if (isInSynch) {
+  //     updateName(signatureEnricher(config.signature));
+  //   }
+  // }, [config.signature, isInSynch, signatureEnricher, updateName]);
+
+  // useEffect(() => {
+  //   setInSynch(data.name === signatureEnricher(config.signature));
+  // }, [config.signature, data.name, signatureEnricher]);
```

Comment:
:ghost:

Reply:
sorry forgot to push the latest comment (also with tests included)